### ISSUE TITLE
Mac build broken on master

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#8766]
Basically all languages are broken:
```
Undefined symbols for architecture x86_64:
  "_grpc_global_wakeup_fd", referenced from:
      _pollset_global_init in libgrpc_unsecure.a(ev_poll_posix.o)
      _pollset_work in libgrpc_unsecure.a(ev_poll_posix.o)
      _kick_poller in libgrpc_unsecure.a(ev_poll_posix.o)
      _pollset_global_shutdown in libgrpc_unsecure.a(ev_poll_posix.o)
```

**Comment by muxi** [comment]
Blocked on this issue as well.

**Comment by ncteisen** [comment]
closed by #8869
